.github/workflows/main.yml

```diff
@@ -3,6 +3,9 @@ on:
   push:
     branches: [main]
 env:
+  NEXT_PUBLIC_DASHBOARD_URL: "https://dashboard.covid19.outbreak.africa"
```

Comment:
Looks good; but can't we make this envs configurable?

Reply:
I think env vars can only be defined in Workflow similar to the way we're doing here: https://docs.github.com/en/actions/learn-github-actions/environment-variables